stl/src/alias.asm

```diff
@@ -0,0 +1,24 @@
+; weak symbol aliases
+
+; Copyright (c) Microsoft Corporation.
+; SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+; Note that while this is a masm file, it contains no code
+; masm does generate a .text section with zero size
+
+IFDEF RAX ; note: this only works with ifdef, not ifndef
+ELSE
+.MODEL FLAT, STDCALL
+ENDIF
+
+EXTERN __imp_InitOnceBeginInitialize :PROTO :PTR PTR VOID, :DWORD, :PTR DWORD, :PTR PTR VOID
+EXTERN __imp_InitOnceComplete :PROTO :PTR PTR VOID, :DWORD, :PTR VOID

```

Comment:
Are these `:PROTO`s at all useful, given that we're never going to `INVOKE` these symbols, or can we omit?